### DISCUSSION TITLE
"we still do not understand the true evolution of the universe"

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#187] [Metaphysics and Philosophy]
from Alexander Unzicker.  Watch one minute of this if not the whole thing: [https://youtu.be/aD0S1rH8AiE?t=1309](https://youtu.be/aD0S1rH8AiE?t=1309)

**Comment by bgreenman** [comment]
I Enjoyed Unzicker's program. If physics wants to move forward it should put am emphasis on studying philosophy and physics-astronomy programs should recruit scholars outside the main physics paradigm. i also feel there should be a grants program for independent science scholars who work outside mainstream institutions to enable them to have time to develop new ideas.

**Reply by budrap00** [reply]
If physics wants to move forward it has to revert to the study of physical systems, rather than the tending of consensus mathematical models.

**Comment by MJoseph315** [comment]
Given that he has endorsed the ‘liquid metallic hydrogen’ theory of the sun, I’m a little skeptical of Mr. Unzicker.

**Reply by bgreenman** [reply]
i Agree

**Reply by RedshiftDrift** [reply]
@MJoseph315 @bgreenman this [ad hominem argument](https://en.wikipedia.org/wiki/Ad_hominem) is a fallacy.  Isaac Newton endorsed alchemy yet produced a theory of gravitation that is still in use today.

**Reply by bgreenman** [reply]
I Mad a mistake

**Comment by OlliSant** [comment]
The universe has a Steady State, not development, no beginning, no end. No expansion. Static universe, dynamic parts. Static space, dynamic matter: galaxies and forces. The Steady State is the web of galaxies always and everywhere, the dynamics is explosions and clouds locally.

This is one possible metaphysics.

**Reply by bgreenman** [reply]
An interesting alternative possibility that should be investigated

**Reply by AndriiKyryl** [reply]
But this "metaphysics" directly contradicts physics, where there is _always_ an irreversible degradation of energy, at any scale (the "second law"). And this degradation is generally not small, it is essential. It is the universal content of "development", which is aging. Without it, there could be no perceivable time (and actually anything else either). So, those "eternal" solutions contradict everything, all observations. This does not mean, of course, that the only possible development way is something like the Big Bang. In reality, it is also in strong contradiction to the generalized second law: just a rough linear-mechanistic simplification of the intrinsically complex interaction dynamics. There are other, properly complex development possibilities (see e.g. [https://arxiv.org/abs/physics/0510240](https://arxiv.org/abs/physics/0510240)).

**Reply by bgreenman** [reply]
There could be a means to recycle degraded energy at the quantum level, but we still don't have the full answer of how the particlin the quantum vacuum wink out into it.

**Reply by RedshiftDrift** [reply]
@AndriiKyryl  I agree that the thermodynamic *"development that shows aging"* is what we perceive as *time*.  This can actually be used as a *definition* of time[^1].  However, thermodynamics only applies to **closed** systems (or approximations to a closed system - e.g. a beaker, a steam engine, a laboratory, Earth's surface, the solar system, a galaxy) where the exchange of energy/entropy with the 'exterior of the system' can approximately be neglected.  And even if the Big Bang model considers closed systems, it is also in contradiction to the generalized second law, as you show.

Thermodynamics says nothing about open systems such as the observed universe, which has no boundary as far as we can see (i.e. larger telescopes have always detected more distant galaxies, there is no evidence of a limit yet).  This is why when we consider a larger volume of space, the less we see changes in the universe[^2], giving less of an appearance of *time*.  No observation shows cosmological time in the distant universe, in fact observations that do not depend on the underlying cosmological model show the opposite[^2].

These "boundless solutions" don't contradict any physics because cosmological distances are beyond the validity range of thermodynamics.  @OlliSant is right about this, the observed distant universe is not seen to evolve, and therefore, shows no perceivable time.

[^1]: The formal definiton of a clock describes three necessary elements: an ***oscillator***, a ***counter***, and an ***indicator***.  The counter is the thermodynamic element that 'ages' with increasing entropy, and the indicator 'shows' aging.
[^2]: Two examples among many: "No Redshift Evolution of Galaxies’ Dust Temperatures Seen from 0 < z < 2" [https://doi.org/10.3847/1538-4357/ac6270](https://doi.org/10.3847/1538-4357/ac6270); and "The MOSDEF Survey: The Evolution of the Mass–Metallicity Relation from z = 0 to z ∼ 3.3*" [https://iopscience.iop.org/article/10.3847/1538-4357/abf4c1](https://iopscience.iop.org/article/10.3847/1538-4357/abf4c1) where they see no evolution of the mass-metallicity relation and conclude *"the processes governing the smooth baryonic growth of galaxies via gas flows and star formation* ***hold in the same form over at least the past 12 Gyr"***.

**Reply by bgreenman** [reply]
Statemen is correct

**Reply by AndriiKyryl** [reply]
@RedshiftDrift Thank you for the detailed answer. However, the general explanation you propose should assume a particular input of low-entropy, high-potential energy/substance somewhere beyond any final or now observed part of the much larger, "infinite" Universe. What could it be, and where could it happen? Thinking about it, it would be a contradictory possibility. Or if the entropy growth reparation can happen more or less everywhere, then again, what could it be, and how can it be observed? The well-known matter-creation assumption in a classical steady-state model looks experimentally doubtful...

The permanent energy and matter recycling you seem to suggest in your model (I'm not sure about the details) would inevitably result in a progressive irreversible energy degradation in each cycle, according to the same universal principle (by the way, I provide its consistent dynamic substantiation in terms of causally complete interaction description). So again, we would need an omnipresent (?) source of compensation for this inevitable entropy growth, if we want an eternal Universe existence. It's true, however, that the energy degradation towards a quasi-dead (finite, observable) Universe can take a very long time, much longer than the standard time-scale order of 10 Gy.

Apart from these entropy-growth problems, while thinking about infinite and eternal Universe, it occurred to me that such "unlimited" Universe structure leads to the "infinitely contrast/hard" version of the Fermi paradox: its every solution for a finite in space and time Universe, is asymptotically "killed" by limitless measures of the latter, with the only remaining possibility that we are (practically) alone in that infinite Universe. What a mind-boggling horror!) Anyway, this is a less rigorous idea.)

**Reply by bgreenman** [reply]
We are only beginning to study possible energy sources in the universe. The modern idea Of Galaxies beyond the Milky Way is barely a century old, it will take time to iron out new ideas

**Reply by RedshiftDrift** [reply]
The basis of my argument is that thermodynamics does not work at cosmological scales.  We cannot therefore use thermodynamics in a discussion about cosmology.

>... you propose should assume a particular input of low-entropy

No, thermodynamics does not apply, there is no need for low-entropy input.

> how can it be observed?

We have not observed anything beyond the limit of our observations.  This is a metaphysical question.

> The well-known matter-creation assumption in a classical steady-state model looks experimentally doubtful...

I ***don't*** support a **steady-state model**.  I am discussing a ***static model*** where energy is conserved.

> would inevitably result in a progressive irreversible energy degradation in each cycle

No, energy is conserved and entropy has no meaning.  There is no degradation of anything, thermodynamics does not apply.

> ... while thinking about infinite and eternal Universe

No finite being can comprehend infinity.  This would require an infinite intelligence and memory, which is a physical impossibility.

**Reply by AndriiKyryl** [reply]
@RedshiftDrift OK, I understand your arguments. You prefer the formal inapplicability of a final-system law to the infinite system. But one question still remains. If the degradation of energy is applicable to any finite, practically closed part of the Universe, while being irrelevant for the infinite Universe (by the way, you seem to infer its _constant_, no-time entropy only empirically, without any theoretical substantiation), then where is a border/transition between those two qualitatively different states of the (ever larger) system? And then again, difficult not to wonder why and how it happens concretely that all the ever larger finite systems should degrade energetically and then in the limit they pass to qualitatively different, no-degradation behavior.
I would add that I am talking about a "generalized" second law, which does not need statistics or thermodynamic equilibrium (for simplicity, we could consider a qualitative fact of energy degradation, due to the inevitable growth of dynamic randomness, or chaos).

**Reply by RedshiftDrift** [reply]
This "border/transition" between 'small closed systems' and 'large open systems' happens when the material parts of the system can no longer move across the size of the system within the lifetime of those parts.

*Example of a small closed system:* the water molecules inside a steam engine (1 km/s) can move a distance comparable to the size of the steam engine in 100 ms, which is much smaller than the lifetime of the water molecule (stable unless dissociated by e.g. red-hot iron).  Thermodynamics works well in that regime, it was actually developed for steam engines.

*Example of a "border/transition" system:* stars in a galaxy (300 km/s) can move a distance comparable to the size of the galaxy in one rotation time (200 million years) which is comparable to the lifetime of a small star (100 million years).

*Example of a large open 'system':* galaxies in a galactic supercluster (3000 km/s) potentially could move a distance comparable to the size the supercluster, but gravitational close-encounters diffuse the galaxies.  It would take galaxies much more than 50 billion years to make it across a supercluster, much longer than the lifetime of a galaxy that gets destroyed by collisions, quenching, instabilities, etc.

The supercluster can no longer be considered a system.  It does not degrade energtically, instead it exchanges energy via light between its material components and other superclusters.  This is the beginning of a transition to the ["timeless" regime](https://github.com/orgs/a-cosmology-group/discussions/187#discussioncomment-7744951) discussed above.  Even star-light travelling across the open-system gets scattered and redshifted down to a microwave background after redshift z = 1000, which corresponds to [a distance of 90 Gly](https://cosmology.info/code/marmet_l/cosmology-calculators.html).  The energy from the degraded radiation is not lost, it provides the energy to [return intergalactic heavy elements back to hydrogen](https://github.com/orgs/a-cosmology-group/discussions/144#discussioncomment-7719752).

Any concept of entropy or a 'generalized second law' implies a direction of time and change.  But time starts to lose its physical significance beyond galaxy clusters: no clock (counter) can survive long enough to measure that time.  Without a clock, 'change' cannot be observed and loses its physical meaning.  As a physicst, this is where my science ends - beyond that we are talking metaphysics, philosophy and religion...

**Reply by sahil5d** [reply]
@RedshiftDrift 

> I am discussing a static model

'static' does have a specific meaning in cosmology, but for popular audiences the word can cause a hiccup in thought, because 'static' typically means "lacking in movement, action" https://www.google.com/search?q=define+static. What do you think about using 'dynamic'? Like 'dynamic cosmology', 'dynamic equilibrium cosmology'.

**Reply by bgreenman** [reply]
Idea worth investigating

**Reply by RedshiftDrift** [reply]
@sahil5d you can explain to the public that 'static' refers to Euclidean space (no action of space), as opposed to the 'expanding space' of Big Bang cosmology.  ***"Static"*** as in:
- no Big Bang,
- no Inflation,
- unchanging laws of physics,
- unvarying fundamental constants,
- trustworthy,
- reliable,
- steadfast,
- etc.  That's what popular audiences should be told.

Popular audiences are interested in 'click bait': explosions, relativistic jets, colliding galaxies, black holes, demolition derbies... what's another hiccup going to do?  How about showing them the real universe which maintains itself against the possibility of falling into total chaos, the "forces of good" standing up to the "forces of evil" for at least 30 billion years (as far as science can tell).  That's better than a Star Wars movie.

There are many more cosmologies than words we can use to describe them succinctly.  Since it is clear that galaxies and planets are 'dynamic', the word to describe a cosmology should refer to what is special about that cosmology, and especially, what makes it different from mainstream cosmology.  As budrap00 pointed out, my misanthropic view doesn't incite me to conform to public opinion.  If 'static' is not exciting enough, popular audiences can remain in their ignorance.  Whatever it is called, “a rose by any other name would smell as sweet.”

P.S. [Andrei Kirilyuk](https://vixra.org/author/andrei_p_kirilyuk) proposes a 'dynamic' cosmology to describe a dynamic interaction process of interacting fields.  Popular audiences have certainly never heard about it.

**Reply by sahil5d** [reply]
'static' as no Big Bang, no Inflation, unchanging laws of physics, unvarying fundamental constants -- that makes sense to me.

But, and I should have mentioned this earlier, there is a another small issue in '*st*atic' sounding like '*st*eady *st*ate' (Hoyle) which is wrong, and big bang cosmologists often use "big bang vs steady state" as a false binary.

The fewer hiccups the better, because popular audiences comprehending ACG's arguments will mean this scientific revolution happens at the speed of internet diffusion instead of academic discussion.

**Reply by js2zonde** [reply]
> But this "metaphysics" directly contradicts physics, where there is _always_ an irreversible degradation of energy, at any scale (the "second law"). And this degradation is generally not small, it is essential. It is the universal content of "development", which is aging. Without it, there could be no perceivable time (and actually anything else either). So, those "eternal" solutions contradict everything, all observations.

Well, it seems to me that observation of redshift contradicts the "second law". Whatever mechanism is responsible for redshift it absorbs energy indiscriminately (as seen in [GW170817](https://en.wikipedia.org/wiki/GW170817) event). We have this heatsink in intergalactic space that allows [dissipative systems](https://en.wikipedia.org/wiki/Dissipative_system) to run. Even gravity wouldn't work as it needs to dump binding energy without getting it all back from the outside.
So no, I don't think that second law of thermodynamics is relevant at cosmic scale. It's useful for steam engines and other dissipative systems here on Earth where we can imagine like some of them are closed systems when actually they are not.

**Comment by bgreenman** [comment]
I support Redshiftdrift's version of a static model in which energy is conserved

**Reply by AndriiKyryl** [reply]
Energy is always conserved, in any model, but normally it tends to _irreversibly_ change its _form_, from "potential" (fuel) to "thermal" (radiation and ashes).

**Reply by bgreenman** [reply]
correct

**Comment by OlliSant** [comment]
The problem is the whole and parts. You cannot understand the whole universe without understanding rightly the parts. And not the parts without right understanding of the whole universe (Whole and parts in Hegel).

Obviously we don't yet have the right understanding of the whole. It is in some kind of Steady State, not the observable universe, not any smaller "universe". Just the biggest totality is in Steady State. And when we go to the bigger and bigger universe, somewhere we meet the biggest one. It has no outside, there is all that exists.

**Reply by OlliSant** [reply]
Universe is in Steady State. That theory is not wrong as such. And the space is static too. The matter is dynamic, the parts of the universe. But matter, time and space are relative. What is coming more, is time and the movements of matter, energy. 

The theory of the universe is missing. We don't have it. Because this difference between the whole and parts in cosmology has not been made in any theory. And sound philosophy is missing. Physics we have enough, and in a wrong bases. And mathematics in the air, not fitting the reality of the universe.

But we seem to be in the treshould of getting better theories and physics and mathematics and philosophy of the universe.

**Comment by LifeIsStrange** [comment]
we still do not understand the true evolution of the universe.
This statement is too vague to derive any insight or common understanding.
The true qualifier serve btw no purpose as in a no true Scotsman.

The universe chemical and morphological evolution is highly constrained, we now have very accurate metallicity laws than span the cosmic lifespan overall.
Of course with a few select exceptions.

**Comment by OlliSant** [comment]
I mean the whole universe, not the local subuniverse. You don't understand that difference. The understanding of the whole universe is vague, it must be vague, because it is vague and will be vague for a long time. And its follows that also the understanding of the local subuniverse is wrong or vague. Better to be vague, and not wrong as now BB is. 
  
  
   
    10.03.2026 15.00 EET LifeIsStrange ***@***.***> kirjoitti:
   
   
     
   
   
     
   
    
   we still do not understand the true evolution of the universe.
    This statement is too vague to derive any insight or common understanding.
    The true qualifier serve btw no purpose as in a no true Scotsman.
   The universe chemical and morphological evolution is highly constrained, we now have very accurate metallicity laws than span the cosmic lifespan overall.
    Of course with a few select exceptions.

**Reply by RedshiftDrift** [reply]
The word 'universe' describes the ensemble of everything.  It cannot be qualified with 'whole' as in "whole universe" since by definition it already describes everything (material, relational, etc.)  There cannot be anything "outside the universe" since by definition this is also included in the universe.

Therefore, the terms 'multiverse', 'local universe', and 'subuniverse' are meaningless.

The universe is not a scientific concept since it cannot be observed.  'Universe' is a philosophical concept, and I agree with "understanding the universe must be vague" to avoid the logical fallacy of self reference.  Philosophically, the universe "is", which is a timeless state.  There is therefore no possible evolution of the universe since an evolved state would not be included in the universe, which would contradict the definition of the word 'universe'.  There is therefore no "true evolution of the universe" to understand since the universe cannot evolve by definition.

The only possible evolution is the life cycle of objects inside the universe, such as galaxies and humans, which are born, grow, and die.

**Reply by HanDeBruijn** [reply]
Our version of the Variable Mass theory (VM) has the assumption that the universe is infinite in space and time (**not** in space-time). Because of [Russel's Paradox](https://en.wikipedia.org/wiki/Russell%27s_paradox) and [_Infinitum Actu Non Datur_](http://www.alternatievewiskunde.nl/MijnBoek/oneindig/index.htm) this effectively means that [**"There is no universe"**](https://medium.com/@petergleeson1/there-is-no-universe-cec835dc575f) at all; in the sense that there would exist something that contains everything. Our _infinite universe_, according to Gauss' dictum, is just _a way of speaking_.

**Reply by RedshiftDrift** [reply]
Rutherford reportedly told his researchers: "Don't let me catch anyone talking about the Universe in my department".  Feynman also dislike the term.
Edit: http://arxiv.org/abs/astro-ph/0009020 Unfortunately I can't find the original source for all this...

Russel's Paradox is about something different - having a set that contains all mathematical sets.  The physical universe does not necessarily contain every mathematical object (like Russel's set $\mathcal R$), it only contains every *physical* objects and their relationships.

**Reply by OlliSant** [reply]
We need a totally new discussion of the "Universe". It is not clear what we mean by the world, how it is used in cosmology, and what it is in the cosmology and in the reality. My thoughts are only preliminary. All again! Everything! With philosophers and astronomers and physicists. Totally new analysis.

And new discussion of the method of the science. Science is theories, and we form theories by reason and observations, and we test theories by reason and observations. Both are always there, not only the empirical science. Then we change theories, when needed. Now we must change the GR.

When we use the right meaning of the "universe", there is no beginning or expansion, they are meaningless.

**Comment by RedshiftDrift** [comment]
C. Wheeler et al., “How Invisible Stellar Halos Bias Our Understanding of Ultrafaint Galaxies,” ApJ, vol. 995, no. 2, p. 162, Dec. 2025,
 doi: [10.3847/1538-4357/ae1698](https://doi.org/10.3847/1538-4357/ae1698).

"This suggests that attempts to use mass estimates of ultrafaint galaxies to constrain dark-matter physics or to place constraints on the low-mass threshold of galaxy formation must take into account surface brightness limits or risk significant biases."

**Comment by OlliSant** [comment]
Universe does not move. No evolution. When we think the biggest entity, it has  a steady state, Parts explode and develop, not the universe. It exists in a different way as other things. But it exists. Anyway as a concept, that is necessary in cosmology, in the theory of cosmology. No beginning, no expansion, they are meaningless.

**Reply by OlliSant** [reply]
More in my totally New Homepage www.santavuori.com same name connects there. Hope comments.

**Reply by RedshiftDrift** [reply]
Very nice website!  I'll add a link to it on [cosmology.info](https://cosmology.info/essays/) in the description of your essay.

**Reply by OlliSant** [reply]
Fine!

**Reply by js2zonde** [reply]
> Universe does not move. No evolution. When we think the biggest entity, it has a steady state, Parts explode and develop, not the universe. It exists in a different way as other things. But it exists. Anyway as a concept, that is necessary in cosmology, in the theory of cosmology. No beginning, no expansion, they are meaningless.

"Universe" is just a word that could mean anything. Should I guess? Is it what you mean with "biggest entity" that consists of parts? But then you say that there are "other things". So "other things" are parts of the universe or are they entities that exist in parallel to the universe.
If I would imagine concept with no evolution, no motion I would imagine infinite space and infinite time, but philosophical concepts of space and time, not the physical interacting spacetime of general relativity. But I am not sure if that is what you mean.
Would you care to elaborate?

**Reply by OlliSant** [reply]
There are no parallel universes. Only parallel part-universes to other part-universes. There is only one universe, no outside at all, or other universes. This is the right meaning of the universe as Louis said. But there can be other part-universes to part-universes. Now many of us say that they are "universes". But they are not, wrong way to use the word "universe".

When we go from physics and astronomy to the universe, we go in the same moment to philosophy, to metaphysics. And this field is not handled at all in modern philosophy. Cosmologists, philosophers, astronomers and physicists must together make a totally new study of "universe", "observable universe", super-clusters, space of the universe, time of the universe, energy and matter of universe, "world" and so on. Many words must be handled again. The beginning of the "universe", the expansion of the "universe". And so on.

What is the relation of physics and metaphysics in cosmology? Nobody knows! There is no study. We must do it. My thoughts are preliminary. How we use these terms in BB? How in Multiverse theories? How in Steady State theories? How in Cyclic theories?

What we mean by "universe"? Is there such a thing, the "universe"? When we define the word, there we have the theory of the universe. It is the existing eternal and infinite space and matter around us. It exists differently as any other thing, it is itself the biggest possible space and not inside a bigger entity as other things exist. No more space or matter is coming from anywhere.

**Reply by AndriiKyryl** [reply]
> What we mean by "universe"? Is there such a thing, the "universe"? When we define the world, there we have the theory of the universe.

Yes, the last sentence is true, and it includes the correct definition of the "universe".

Namely, the uni-verse is the _physically unified_ structure of the cosmos, including all its sufficiently observable parts, for which we can confirm the same fundamental physical properties (mainly those of space, time, particle properties and related observable features). It is usually taken for granted or assumed intuitively. It is close to "observable universe" but with this exact understanding of the same, unified physical properties and laws. Once we observe a confirmed difference in fundamental properties within the "observable universe", it may witness the presence of "something else", different from the universe thus defined, maybe the "edge of the universe" or an entrance manifold to another universe.

Of course, to be truly rigorous, this definition of the universe would need the causally specified and unified origin of fundamental physical entities (space, time, particles, interactions), properties and laws, which is absent in current theories. I provide such origin in my theory, but apparently it  is not of any interest to other dwellers of the Earth. Nevertheless, even without that total rigor, the above definition of the universe looks reasonable and experimentally efficient.

**Reply by OlliSant** [reply]
We don't see, observe the whole universe, so the empirical study of the universe alone is futile. Must have rational thought and empirical study together. Then, when we have theories, we can test them in the science. And the testing too is both rational and empirical. We can make science and philosophy of the universe, but there must be theories and then testing of them. If we only prove GR and BB, this is not science. We simply make theories and test them. Now it is forbidden almost, because we must believe in BB or be solely empirical. That is not science or sound philosophy.

In philosophy we must make better definitions on the "universe". Nobody in philosophy is making this now because everybody believes in BB and has leaved cosmology to physicists. That is a big mistake. In BB and in other theories of the universe there is always wrong definitions of the universe and all the other concepts. When we change the paradigm, we change the definitions of the concepts to better correspond with the reality, which is unknown, not known.

**Reply by OlliSant** [reply]
Where is your theory?

**Reply by AndriiKyryl** [reply]
See e.g. https://arxiv.org/abs/physics/0510240 for cosmological applications or https://hal.science/hal-02489867 for an updated and more general exposition of the same theory. All applications and references can be found at https://sites.google.com/site/unifiedcomplexity/.

**Reply by HanDeBruijn** [reply]
There is NO Theory of Everything. Period.

**Reply by AndriiKyryl** [reply]
Even more, there is no consistent theory of _anything_, starting already from the isolated electron, the simplest stable object. This is because "our best minds" have decided that it should remain like that, period. No way to try, any math or logic is irrelevant. After all, the physically unified everything always exists, and our best minds continue to obtain their nice and quite real profits, for saying that everything is completely mystique and cannot be not only unified but even logically understood, as follows from their purely abstract Ptolemaic constructions. So why bother, everything is already done in their best of all possible worlds.

**Reply by OlliSant** [reply]
Kyryl, your theory seems to be fine, but too technical for me to understand. You must popularize it first

**Comment by IngAlopez** [comment]
> De Alexander Unzicker. Vea un minuto de esto, o incluso todo: https://youtu.be/aD0S1rH8AiE?t=1309

https://zenodo.org/records/19645644

Nature is the greatest teacher there is; it has enlightened humanity since ancient times. However, we often ignore it because we believe ourselves to be wise. Attached is an impact proposal based on this idea...

**Reply by RedshiftDrift** [reply]
As mentioned by Unzicker and de Grasse Tyson, people take two things that are not well understood: ["fantasies about the first seconds of the Big Bang" and fantasies about "high energy particle physics"](https://youtu.be/aD0S1rH8AiE?t=1339), and use one to explain the other.

The [Sexagesimal Cosmological Model](https://zenodo.org/records/19645644) does even worse.  It uses mathematics as if it were a "physical science" (math is a formal language, not a science) and builds a fantasy based on the magic number "60" to explain two other fantasies that are not well understood.  In a similar vein, William Cox proposes a [cosmology from 'base-16 mathematics'](https://www.amazon.com/Universal-Mathematics-Language-Artificial-Intelligence-ebook/dp/B0D8WLC243).

There are plenty of numbers to choose from, in case you like being [lost in math](https://www.amazon.ca/Lost-Math-Beauty-Physics-Astray/dp/0465094252) forever...
